pkgs/tools/security/sscep/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, libtool
+, pkg-config
+, m4
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sscep";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "certnanny";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0q25z4s5sq97yrpv54wsrxr3vz9cd31ci04ffj6znb42scw50p62";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+    pkg-config
+    m4
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  preConfigure = "./bootstrap.sh";
+
+  meta = with lib; {
+    description = "Command line client for the SCEP protocol";
+    homepage = "https://github.com/certnanny/sscep";
+    license = [ licenses.bsd3 licenses.openssl licenses.vsl10 ];
```

Comment:
[From upstream notice](https://github.com/certnanny/sscep/blob/master/COPYING#L2-L25) it looks more like BSD-2.

And I'm not sure we need to list the OpenSSL license here, it's just linked.
I also can't find what part of the code is under the VSL-1.0 license. It's included in the COPYING notice but [their own spec file](https://github.com/certnanny/sscep/blob/master/scripts/sscep.spec#L10) does not mention it.